server/service/GithubService.js

```diff
@@ -246,6 +249,7 @@ export class GithubService {
     // check if it's there already
     let hooks = await this.fetchPath('GET', path, null, accessToken)
     let existing = hooks.find(h => h.config.url === hook_url)
+    // MIssing CHECKS
```

Comment:
what kind of checks are missing?

Reply:
This was a past comment, when I had to go through the whole chaining of method calls before knowing exactly where the 403 Error should be handled.